### PR TITLE
Include ger and yurt, as well as man_made

### PR DESCRIPTION
**Pull request by habi**

### CODE REVIEW

**Comment by habi** [comment]
Related to https://github.com/osm-fr/osmose-backend/issues/1234

**Comment by habi** [comment]
As mentioned in the linked issue, I have *no* experience with SQL and just tried to estimate where I can expand on the list of things to consider...
Any input/corrections are welcome!

File: analysers/analyser_osmosis_building_shapes.py [hunk 1]
```diff
@@ -36,8 +36,8 @@
     polygon_proj IS NOT NULL AND
     area / ST_Area(ST_MinimumBoundingCircle(polygon_proj)) > 0.95 AND
     ST_MaxDistance(polygon_proj, polygon_proj) > 5 AND
-    tags - ARRAY['created_by', 'source', 'name', 'building', 'note:qadastre'] = ''::hstore AND
-    tags->'building' NOT IN ('hut','slurry_tank')
+    tags - ARRAY['created_by', 'source', 'name', 'building', 'man_made', 'note:qadastre'] = ''::hstore AND
+    tags->'building' NOT IN ('hut', 'ger', 'yurt', 'slurry_tank')
```
Comment (frodrigo) [inline analysers/analyser_osmosis_building_shapes.py:40]:
```suggestion
    tags->'building' NOT IN ('hut', 'ger', 'yurt', 'slurry_tank') AND
    tags->'man_made' NOT IN ('water_tower', 'reservoir_covered', 'wastewater_plant', 'storage_tank', 'windmill', 'dovecote')
```

**Comment by frodrigo** [comment]
Merged, not deployed yet.

**Comment by frodrigo** [comment]
Deployed

**Pull request merged**